src/client/apps/edit/components/admin/components/featuring/index.tsx

```diff
@@ -3,16 +3,22 @@ import AutocompleteListMetaphysics from "client/components/autocomplete2/list_me
 import React from "react"
 import { FeaturingMentioned } from "./featuring_mentioned"
 
-export const AdminFeaturing = () => {
+interface AdminFeaturingProps {
+  isAdmin: boolean
+}
+
+export const AdminFeaturing = (props: AdminFeaturingProps) => {
```

Comment:
We'll want to use react-specific types for this (and elsewhere, if there are changes):

```tsx
export const AdminFeaturing: React.FC<AdminFeaturingProps> = props => { ... }
```

Reply:
changed. thanks @damassi .